### PR TITLE
Dialogflow: Added sentiment analysis to query result payload

### PR DESCRIPTION
**Pull request by beeme1mr**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://dialogflow.com/docs/reference/api-v2/rpc/google.cloud.dialogflow.v2beta1#queryresult
- [X] Increase the version number in the header if appropriate.

### CODE REVIEW

**Review by dyladan — APPROVED** [review]
This looks good to me and I can confirm the payload and documentation both match this type. I'm the original contributor of these types :)

edit: see later review

**Comment by dyladan** [comment]
This has the unowned tag, but I originally contributed these types. Is there a way I can take ownership of this? Or does "owner" refer to the owner of the package being referenced by the types?

File: types/dialogflow/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for dialogflow 0.6
+// Type definitions for dialogflow 0.7
```
Comment (dyladan) [inline types/dialogflow/index.d.ts]: This is the version of the package being typed. Still at 0.6

**Review by dyladan — CHANGES_REQUESTED** [review]

**Review by dyladan — APPROVED** [review]
Better thanks

**Comment by dyladan** [comment]
@RyanCavanaugh Any idea why @typescript-bot doesn't recognize me as the author?

**Comment by RyanCavanaugh** [comment]
@dyladan looks like the CODEOWNERS file is out of date. I'll have someone update it

**Review by dyladan — APPROVED** [review]
Trying to add another review after CODEOWNERS file change

**Pull request merged**